forms.py

```diff
@@ -38,6 +42,7 @@ class PewSheetForm(FlaskForm):
     anthem_title = StringField('Anthem')
     anthem_composer = StringField('Anthem composer')
     anthem_lyrics = StringField('Anthem lyrics', widget=TextArea())
+    anthem_translation = SelectField('Anthem Translation', choices=translations)
```

Comment:
I don't know if Flask allows nesting of forms but it might be nice to group these together